app/lib/analyzer/pana_runner.dart

```diff
@@ -22,8 +22,8 @@ import '../shared/tool_env.dart';
 final Logger _logger = Logger('pub.analyzer.pana');
 
 @visibleForTesting
-final pedanticAnalysisOptionsYaml = File(p.join(static_files.resolveAppDir(),
-        'lib', 'analyzer', 'pedantic-analysis_options.1.8.0.yaml'))
+final defaultAnalysisOptionsYaml = File(p.join(static_files.resolveAppDir(),
```

Comment:
Could we instead load this directly from package:lints with a resolvePackageUri?

Reply:
I'll do that in a follow-up PR.